app/views/admin/versions/_object_desc_and_link.html.haml

```diff
@@ -1,15 +1,19 @@
+- conference = Conference.find_by(id: version.conference_id)
+- conference_short_title = conference.try(:short_title) || 'deleted_conference'
+- conference_title = conference.try(:title) || 'Deleted Conference'
+
 - if version.item_type == 'UsersRole'
   - users_role = get_version_object(version)
   = 'role'
-  = link_to users_role.role.name, admin_conference_role_path(conference_id: Conference.find(version.conference_id).short_title, id: users_role.role.name)
+  = link_to users_role.role.name, admin_conference_role_path(conference_id: conference_short_title, id: users_role.role.name)
```

Comment:
Hmmm I didn't properly catch this before.

Here it is possible that we will create a link that does not exist. That does not sound right :/
If there is no conference record there is no point in linking anywhere, because we can't visit that link anyway.